scripts/modifyConfigIfRCVersion.js

```diff
@@ -2,9 +2,10 @@ const fs = require('fs');
 
 if (process.env.TRAVIS_TAG) {
   const ISTESTVERSION = /^v?[0-9]+\.[0-9]+\.[0-9]+-[rR][cC]$/.test(process.env.TRAVIS_TAG || '');
-  const packageJson = JSON.parse(fs.readFileSync('package.json'));
   if (ISTESTVERSION) {
```

Comment:
I think camalCase would be more easy to read.

Reply:
fixed.